doc/codingGuidelines.md

```diff
@@ -156,8 +162,8 @@ All included public header files from outside and inside the project should be `
 
 Private project’s header files should be `#include`'d using the file name when in the same folder. Private headers may live in sub-directories, but they should never be included using "._" or ".._" as part of a relative paths. For example:
```

Comment:
One comment I heard about this line was to make it more explicit about the fact we are using relative paths, i.e. not a full path as above.

Reply:
Will make that change...

Reply:
See: https://github.com/Autodesk/maya-usd/pull/400/commits/dd99332698132e11a7416327a4089502ed7dd787